src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $weak-day, $index in $start-day {
+    &--start-day-#{$weak-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
```

Comment:
The SCSS variable '$start-day' used in the 'weak-day-lentgh' mixin is not defined in this file. Ensure that it is defined in 'variables.scss' or within this file if it is missing.